js/Events/newEvents.js

```diff
@@ -0,0 +1,79 @@
+const addEventsNew = function() {
+    const dataObject = {};
+    const buttonEval = document.getElementById('button1');
```

Comment:
the button id is not really representative to the button, is it?

Reply:
why not? i want to access it

Reply:
not being representative it means that the id text you chose does not represent the button itself. You can be a bit more verbose so that your code is easier to read and more intuitive

Reply:
Oh sorry, I misunderstood. I will rename them